### PR TITLE
Re-enabling tests that were disabled due to VS 2017 migration …

### PR DESCRIPTION
**Pull request by ahsonkhan**
...(due to resx files).
- Also Renaming some tests that had name conflicts with other tests.

The tests that are failing are from System.IO.Pipelines.Tests caused by the following PR #1166 which got checked in while these tests were disabled.

### CODE REVIEW

**Comment by ahsonkhan** [comment]
cc @KrzysztofCwalina, @shiftylogic, @crummel

**Comment by ahsonkhan** [comment]
@dotnet-bot test please

**Comment by KrzysztofCwalina** [comment]
@dotnet-bot test this please

File: src/System.Collections.Generic.MultiValueDictionary/System.Collections.Generic.MultiValueDictionary.csproj [hunk 1]
```diff
@@ -17,6 +17,7 @@
     <PackageProjectUrl>https://github.com/dotnet/corefxlab</PackageProjectUrl>
     <PackageLicenseUrl>http://go.microsoft.com/fwlink/?LinkId=329770</PackageLicenseUrl>
     <PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
+    <RootNamespace />
```
Comment (davidfowl) [inline src/System.Collections.Generic.MultiValueDictionary/System.Collections.Generic.MultiValueDictionary.csproj]: Remove?

File: src/System.IO.Pipelines/System.IO.Pipelines.csproj [hunk 2]
```diff
@@ -20,6 +20,7 @@
     <GenerateAssemblyConfigurationAttribute>false</GenerateAssemblyConfigurationAttribute>
     <GenerateAssemblyCompanyAttribute>false</GenerateAssemblyCompanyAttribute>
     <GenerateAssemblyProductAttribute>false</GenerateAssemblyProductAttribute>
+    <RootNamespace />
```
Comment (davidfowl) [inline src/System.IO.Pipelines/System.IO.Pipelines.csproj]: Remove

File: src/System.Text.Formatting.Globalization/System.Text.Formatting.Globalization.csproj [hunk 3]
```diff
@@ -17,6 +17,7 @@
     <PackageProjectUrl>https://github.com/dotnet/corefxlab</PackageProjectUrl>
     <PackageLicenseUrl>http://go.microsoft.com/fwlink/?LinkId=329770</PackageLicenseUrl>
     <PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
+    <RootNamespace />
```
Comment (davidfowl) [inline src/System.Text.Formatting.Globalization/System.Text.Formatting.Globalization.csproj]: Remove

Comment (ahsonkhan) [inline src/System.Text.Formatting.Globalization/System.Text.Formatting.Globalization.csproj]: The tests fail if I remove this. I grabbed this change from here: PR #1171 

The same applies in other projects that use resx.

Failed   System.Text.Formatting.Globalization.Tests.NonInvariantCultureTests.CustomCulture
Error Message:
 System.Exception : resource missing
Stack Trace:
   at System.Text.Formatting.EncodingProvider.CreateEncoding(String localeId) in C:\Users\ahkha\Documents\GitHub\Fork\corefxlab\src\System.Text.Formatting.Globalization\System\Text\Formatting\FormattingDataProvider.cs:line 27
   at System.Text.Formatting.Globalization.Tests.NonInvariantCultureTests.CustomCulture() in C:\Users\ahkha\Documents\GitHub\Fork\corefxlab\tests\System.Text.Formatting.Globalization.Tests\BasicTests.cs:line 16

**Comment by ahsonkhan** [comment]
@dotnet-bot test this please

**Comment by ahsonkhan** [comment]
@dotnet-bot test this please

**Comment by ahsonkhan** [comment]
Partially fixes #1155

**Pull request merged**